lib/takeaway.rb

```diff
@@ -0,0 +1,22 @@
+class Takeaway
+
+  def initialize(menu:)
+    @menu = menu
+  end
+
+  def show_menu
+    @menu.print_menu
+  end
+
+  def order_summary(order)
+    order.reduce(0) do |total, line_item|
+      price = @menu.dishes[line_item[:dish]]
+      total + line_item[:qty] * price
+    end
+  end
+
+  private
+
+  attr_reader :menu
```

Comment:
I wasn't quite sure what the benefits were of making the attr_reader for :menu private or if the attribute reader was in fact used anywhere inside the class? All the code in the class seems to refer to the instance variable directly (@menu). I tried commenting out the attr_reader and all the tests still passed, perhaps indicating that the attr_reader isn't required.